frontend/src/lib/api/destructionLists.ts

```diff
@@ -47,9 +47,8 @@ export type DestructionListAssigneeUpdate = {
 };
 
 export type DestructionListItemUpdate = {
-  zaak: string;
   status?: string;
-  zaakData?: Zaak;
+  zaak: string;
```

Comment:
So If I understand correctly we don't have a reference to the zaak here (as zaak is a `string` or should zaak be `Zaak`?

Reply:
Ah sorry, I looked at it again: for the Update operation, you don't need to send all the data of the zaak, just the URL. So it is correct here: zaak is a url!